lib/db/schema/patch-080-081.sql

```diff
@@ -0,0 +1,308 @@
+--
```

Comment:
Should we remove the `capabilities` table/code, and in fxa-auth-server just accept `capabilities` in the incoming device bodies?

Reply:
Some of the existing stored procedures try to pull data from the `capabilities` table, so we can't drop it until we've fully rolled out the updated stored procedures.